---
bibliography: references.bib
format:
  pdf:
    keep-tex: true
    number-sections: false
    include-in-header: latex-header.tex
execute:
  warning: false
editor: source
---

**Manuscript type:** Article

**Title:** Marine invasive species detected from the air

**Authors:** James G. Hagan$^{1, 2}$\*, Fabian Roger$^{3}$, Bram Vanschoenwinkel$^{1, 4}$, Thomas G. Dahlgren$^{5, 6}$, Lena Granhag$^{7}$, Bjorn Kjällström$^{8}$

1.  Community Ecology Lab, Department of Biology, Vrije Universiteit Brussel (VUB), Brussels, Belgium\
2.  Gothenburg Global Biodiversity Centre, Gothenburg, Sweden\
3.  DNAir, Winterthur Switzerland\
4.  Centre for Environmental Management, University of the Free State, Bloemfontein, South Africa\
5.  Department of Marine Sciences, University of Gothenburg, Gothenburg, Sweden\
6.  NORCE Norwegian Research Centre, Bergen, Norway\
7.  Gothenburg Marine Biological Laboratory, Gothenburg, Sweden\
8.  Department of Shipping and Marine Technology, Chalmers University of Technology, Gothenburg, Sweden

**Corresponding author:** James G. Hagan (james_hagan\@outlook.com)

**Data availability:** 

**Code availability:** 

**Key words:** airborne eDNA, biodiversity, monitoring, terrestrial, ocean

## Abstract



## Introduction

One of the major breakthroughs in the field of biodiversity monitoring is the discovery that environmental DNA (eDNA), the genetic material left by organisms in their environment, can reveal the presence, absence, and abundance of taxa across the tree of life. This breakthrough has been especially impactful in aquatic ecology and has allowed researchers to monitor aquatic biodiversity through efficient, minimally invasive means, including DNA-metabarcoding of water samples to assess a wide range of taxa and species-specific real-time PCR assays for targeted species detection. As a result, aquatic eDNA has provided aquatic ecologists with a tool for detecting diverse taxa at large spatial and temporal scales.

More recently, airborne eDNA, genetic material captured directly from the atmosphere, has emerged as a potential tool for biodiversity monitoring. Increasing evidence indicates that air samples can contain DNA fragments from a wide range of taxa and, in some cases, provide information on their relative abundance. These fragments likely originate from diverse biological materials, including skin, hair, bodily fluids, pollen, seeds, and spores. Pioneering zoo-based studies demonstrated that airborne eDNA could detect species both within enclosures and in adjacent areas. Moreover, subsequent field trials have reported partial concordance between taxa identified from air samples and those recorded by established monitoring methods. Together, these results have opened the possibility for airborne eDNA to become a core biomonitoring tool and, as a result, repeat the success of aquatic eDNA.

Interestingly, several airborne eDNA studies have reported detections of strictly aquatic taxa, even though these species were not the primary focus of the work. Such detections have often been dismissed as potential artefacts arising from laboratory or field contamination (e.g. Tournayre et al. 2025 suggest fish DNA may have originated from fish markets). However, there are plausible natural mechanisms by which DNA from aquatic organisms could enter the air, including aerosolisation from breaking waves, wind-driven spray, and the resuspension of water droplets (Ip et al. 2025a; 2025b). These processes raise the possibility that airborne detections may, in some cases, reflect genuine biological transfer rather than contamination.

Indeed, a growing body of evidence supports this view. For example, Klepke et al. (2022) detected nine fish species in airborne eDNA samples collected along the Danish coast, while Sullivan et al. (2023) reported 16 fish genera from airborne samples obtained at a radionuclide monitoring station in northern Sweden. Similarly, Lynggaard et al. (2022) and Clare et al. (2022) identified fish species in airborne eDNA from urban zoos, and Lynggaard et al. (2023) subsequently detected two fish species in samples from a mixed forest. Most recently, in the first national survey of terrestrial biodiversity using airborne eDNA, Tournayre et al. (2025) also detected several species of marine fish in their airborne eDNA samples. Collectively, these findings suggest that the presence of aquatic taxa in airborne eDNA studies may be more widespread than previously assumed.

Recently, two studies by Ip et al. (2025a, 2025b) directly investigated the potential transfer of DNA between freshwater and terrestrial environments. Using paired air and water samples collected above salmon spawning streams, they showed that aquatic eDNA can be reliably detected from passive air samples placed nearby. Their results showed that cross-medium detections are neither random nor artefactual. Instead, the likelihood of detecting a species in air or water depends strongly on its abundance in its native environment, with common taxa appearing consistently across both media and rarer species requiring greater sampling effort to be recovered. Moreover, the temporal dynamics of dominant species were closely mirrored between air and water, indicating that eDNA signals in the two reservoirs are tightly coupled.

Here, we extend this framework to a marine setting. Coastal environments are highly dynamic, with breaking waves, bubble-burst aerosolisation, wind-driven spray, and splash-driven resuspension all capable of injecting aquatic DNA into the atmosphere. Onshore winds may then transport these aerosols inland, where they can be intercepted by airborne eDNA samplers. We leverage these processes to test whether marine-derived eDNA can be consistently detected in air and whether airborne signals reflect true ecological patterns rather than contamination. Specifically, we aimed to answer two questions:

**Question 1:** Are detections of aquatic taxa from air samples reflective of true biological transfer via, for example, aerosolisation or are they simply artefacts of contamination? 

To address this question, we collected paired water and air samples during spring at three coastal sites around Gothenburg, Sweden, that differ in salinity: Nya Varvet (low, brackish), Saltholmen (medium), and Vrångö (high, oceanic). We also sampled at an inland control site located X km from the coast. Using digital PCR (dPCR), we tested these samples for the presence of the Round Goby (*Neogobius melanostomus*). As *N. melanostomus* typically inhabits low-salinity, brackish environments, we expected detections at Nya Varvet and/or Saltholmen but not at Vrångö. The inland control site served as an additional negative control.

As a further test, we conducted two additional sampling rounds in early and late summer at the three coastal sites. These samples, together with the spring samples, were analyzed using dPCR to detect the sea-walnut (*Mnemiopsis leidyi*) an invasive comb jelly which becomes active during the summer. We expected *M. leidyi* to be detectable in summer samples but absent from spring samples.

With this design, if **(1)** detections of *N. melanostomus* in spring are consistent between air and water samples, and **(2)** detections of *M. leidyi* in summer versus spring are also consistent between air and water samples, it would be highly unlikely that positive detections in air are simply due to contamination. Instead, the evidence would support the hypothesis proposed by Ip et al. (2025a, 2025b) that true biological transfer of genetic material occurs from water into air.

**Question 2:** Do we detect similar levels of aquatic species diversity using air and water samples?

To address this question, we conducted DNA metabarcoding using MiFish primers (Miya et al. 2015), which specifically target fish, on the air and water samples collected at Nya Varvet, Saltholmen, and Vrångö during spring. We then compared the diversity and community composition of fish detected across air and water samples. If fish diversity and composition are similar between air and water samples, this would indicate that airborne eDNA has strong potential for surveying aquatic biodiversity.

Together, these analyses provide a critical test of whether airborne aquatic eDNA reflects true biological transfer or contamination, and evaluate its capacity to recover biodiversity patterns comparable to water-based surveys. By comparing air and water samples in a marine environment, our study advances understanding of eDNA ecology and informs the development of airborne eDNA as a complementary tool for aquatic biodiversity monitoring.


## Materials and methods

### Description of the study region

The study was conducted in the coastal region surrounding Gothenburg, Sweden. The area characterized by rocky shorelines, sheltered bays, and extensive intertidal habitats. The region experiences a temperate oceanic climate with a mean annual temperature of around 8°C and a mean annual precipitation of between 800 and 1000 mm. The part of the Kattegat around Gothenburg is influenced by a strong salinity gradient, where saline waters from the North Sea mix with brackish outflows from the Baltic Sea. As a result, surface salinity typically ranges between 15–25 PSU, supporting a mosaic of habitats including kelp forests, eelgrass beds, rocky reefs, and shallow soft-sediment areas.

### Overview of sampling

Sampling took place at four sites in Gothenburg, Sweden: Nya Varvet (NV, Lat: 57.6862350, Lon: 11.8854508), Saltholmen (ST, Lat: 57.6594212, Lon: 11.8372307), Vrango (VR, Lat: 57.5742915, Lon: 11.7758066) and an inland control site (IC, Lat: 57.6695114, Lon: 11.9047494). During the spring (May), at each of these sites (barring the inland control site, IC), we took five water samples and five air samples daily over the course of five days between 14 May and 18 May 2025. At the inland control site, which we set-up to make sure that the air samples were giving us a uniquely coastal signal, we took two air samples: one on 16 May and one on 17 May. Sampling continued through the summer and, at each site barring the inland control site, we took two additional water and air samples (see Figure 2 for an overview).

### Air sampling

To collect airborne eDNA, we deployed a research-grade air-sampling prototype provided by DNAir (https://www.dnair.earth) at each site. Each sampler consists of a 110 mm diameter PVC housing fitted with a filter mount and an electric fan powered by a portable power bank. The fans operate continuously for  *ca.* 24 hours on a single charge. Given that the fan draws approximately X L of air per minute, sampling over a 24-hour period means that more *ca.* Y L of air will pass through the filter.

We prepared the filters by cutting 75 mm diameter disks from non-sterile industrial filter mats (Unifil, [Filter Type]) and sterilised the disks prior to deployment. All preparation took place in a PCR-free laboratory on a workbench decontaminated with a 1:10 dilution of household bleach (0.5% sodium hypochlorite). Filters were immersed in 0.5% sodium hypochlorite solution for 30 minutes, effectively degrading DNA and inactivating microbial spores (Rutala & Weber, 2008; Fukuzaki, 2006). Following decontamination, filters were thoroughly rinsed with Milli-Q water (>3 exchanges, 2 L each) and dried overnight under continuous laminar airflow while mounted on 0.5% sodium hypochlorite-treated stainless steel skewers. Filters were stored in 1 L SafeLoc ziplock bags (Toppits) that had been UV-treated inside and out for 1 hour under the laminar flow hood. Finally, filters were sterilised using a Benchmark UV-Clave (B1450-E V1.0), exposing each side to 254 nm UV for 10 minutes (total dose  *ca.* 600–1200 mJ/cm²). This exceeds the 100–300 mJ/cm² threshold required to eliminate PCR-amplifiable DNA (Champlot et al., 2010; Hofreiter et al., 2001; Cooper and Poinar, 2000). Fully sterilised filters were sealed in the UV-treated bags inside the laminar flow hood until field deployment.

Sampling consists of mounting the sterilised filters to the filter mount using 0.5% sodium hypochlorite-treated tweezers, attaching a fully charged power bank, turning on the fan and letting the fan run for *ca.* 24 hours. After *ca.* 24 hours, the filters are collected and the filter becomes the airborne eDNA sample. Collecting the filters follows the following protocol. First, we switch off the power-bank to stop the fan. Once the fan has stopped running, we remove the rain cover on the front of the air sampler and place it on a plastic surface cleaned with 0.5% sodium hypochlorite and rinsed with Milli-Q water. Next, we open a 10 ml sterile syringe, remove the plunger and place the syringe tube and plunger back into its original plastic bag in preparation for extracting the filter. 

To extract the filter from the air sampler, we carefully cut the filter loose using scissors and tweezers cleaned with 0.5% sodium hypochlorite and rinsed with Milli-Q water. We then place the filter into the 10 ml syringe tube that we had previously prepared. Next, we remove excess air from the filter by putting the plunger back into the syringe tube and compressing the filter. To avoid having to push air through the filter during preservation with ATL buffer, we remove the plunger and and use a needle cleaned with 0.5% sodium hypochlorite and rinsed with Milli-Q water to push the compressed filter up to the base of the syrgine tube. A Luer-lock is then attached to the end of the syringe and 2.5 ml of ATL buffer is carefully dripped onto the compressed filter using a 3 ml syringe. We then place the plunger back into the tube (so that the plunger is now in contact with the compressed filter), remove the Luer-lock adn slowly push the compressed filter towards the end of the syringe tube. As soon as the ATL buffer starts to exit the tip of the syringe, the syringe is closed with a Luer-lock and placed into a clean plastic bag. 

This compressed filter with ATL buffer in a Luer-lock sealed 10 ml syringe constitutes a single air sample. The samples were placed in a 4°C fridge as soon as possible when returning from the field. In addition to the six air samples, we also carried around three filters throughout the field work that were never exposed to the air as negative controls. These were preserved using the same protocol as described above. Once the samples had been transported back to the DNA labs at the Vrije Universiteit Brussel (approximately 3 days after sampling), the samples were frozen at -18°C. The samples were not immediately frozen to avoid freeze-thaw cycles during transport.

### Water sampling

At each site (barring the inland control site, we took a water sample in the water directly adjacent to where the air sampling devices were placed. To do this, we observed the movement of the water to identify upstream areas (where the sample will be taken) and downstream areas. Using a plastic bucket that had been sterilised with 10% commercial bleach and rinsed with Milli-Q water, we thoroughly rinsed the bucket in the downstream area. Using this rinsed bucket, we took at least 1 L of water from the identified upstream area. Using the sampled water, we rinsed a 1 L beaker that was thoroughly sterilised with 10% commercial bleach and rinsed with Milli-Q water. We then placed 1 L of the sampled water into the 1 L rinsed beaker. A sterile 100 ml syringe was then used to suck up 100 ml of the sampled water from the 1 L beaker. We attached a sterile Sterivex filter to this filled 100 ml syringe and slowly pushed the 100 ml of sampled water through the Sterivex filter. We aimed to push the entire 1 L of water through the Sterivex filter and, therefore, repeated this process until 1 L of water had been pushed through the Sterivex. Occasionally, the Sterivex filter became clogged and we were unable to push through the entire 1 L of water. Once 1 L of water had been pushed through the Sterivex (or once the filter was clogged), we removed excess water from the Sterivex filter by shaking it and pushing air through the Sterivex filter using an empty 100 ml syringe. Finally, we added ATL buffer to the Sterivex filter until it the first drop came out. For this, we used a 3 ml syringe that was prepped with 2.5 ml of ATL buffer. Luer-locks were then added to both ends of the Sterivex filter and the Sterivex filter was then placed into a clean plastic bag. As with the air samples, the water samples were placed in a 4°C fridge as soon as possible when returning from the field and frozen at -18°C once they arrived in Belgium. In addition to these five water samples at each site, we also filtered 1 L of commercial mineral water at each site using the same protocol as a negative control.

### DNA extraction

From Hestetun et al. (2023):

"DNA extraction was done at clean NORCE lab facilities in Bergen  (Norway), with lab surfaces and equipment pre-cleaned using  5% bleach solution. Lysis of filtered particles was done inside the  Sterivex filters to minimize contamination and maximize lysis efficiency. Sixty μL of 20 mg mL−1 Proteinase K (QIAGEN) were added to  each thawed filter, which were then incubated at 56°C with gentle  rotation overnight. Lysate was aspirated from Sterivex filters using  sterile 5 mL syringes. One mL of each lysate was taken for DNA purification, while the remaining lysate was archived at −80°C. DNA  purification was conducted using the DNeasy Blood & Tissue kit  (QIAGEN) according to the manufacturer's protocol, with two modifications: (1) Added volumes of RNAse A (100 mg mL−1) and Buffer  AL were adjusted to compensate for increased starting volume of  lysate; (2) Buffer AL-treated lysates were applied to silica spin columns in multiple centrifugation rounds to allow binding of the entire lysate volume. Purified DNA was eluted in 200 μL Buffer EB  (QIAGEN) and divided into one archive aliquot (−80°C storage) and  one working aliquot (−20°C storage)."

### dPCR

#### Round Goby: *Neogobius melanostomus*

Round Goby used the dPCR developed by Green et al. (2025):

From Green et al. (2025)

"To detect the presence of DNA from the target species in the experimental water samples, the dPCR assay developed in this study was used. dPCR reactions were set up in a 26 K dPCR nanoplate in 40 μL reaction volumes, containing 10 μL QIAcuity probe Mastermix 4× (QIAGEN), 2 μL primer-probe assay 20× (Bio-Rad), 10 μL DNA sample, and 18 μL nuclease-free water. The cycling parameters were 95°C for 2 min, followed by 40 cycles of 95°C for 30 s and 64°C for 1 min. Two dPCR reactions were performed for each sample, and the average value was calculated. DNA extracted from N. melanostomus tissue was used as a positive control in the assay, and DNA/RNA-free water was used as a negative control. As the amount of DNA shed correlates with biomass (Rourke et al. 2022), the results of each replicate were pooled and expressed as copy number per replicate fish mass."

#### Sea-walnut: *Mnemiopsis leidyi*



### Metabarcoding

From Hestetun et al. (2023):

"Metabarcoding comprised the MiFish universal fish mt12S rDNA  gene primer pair MiFish-U-F (5′- GTCGGTAAAACTCGTGCCAGC3′) and MiFish-U-R (5′- CATAGTGGGGTATCTAATCCCAGTTTG-3′; Miya et al., 2015). PCR amplification was done in a clean-room UV cabinet with adapter-linked primers using the  KAPA3G Plant PCR kit (KAPA Biosystems) with 35 cycles of 65°C  for MiFish, and 30 cycles of 57°C for 18S primers respectively.  Three (18S) and eight (MiFish) PCR replicates were made for each  sample and subsequently pooled prior to second PCR (library prep).  Purification and quantification after each PCR were done using  Magbio beads (MAGBIO Genomics) and a Qubit 4 fluorometer  (Thermo Fisher Scientific), respectively. Library preparation was  done using equimolar pooled PCR product with Illumina dual index  TruSeq i5/i7 barcodes. Contamination was monitored using field  sampling, extraction, and PCR negative controls. Sequencing was  performed on an Illumina MiSeq instrument using v3 with 300 bp  chemistry at the Norwegian Sequencing Centre (University of Oslo, Norway). The raw data were deposited at the NCBI Sequence Read  Archive (SRA) (18S, SUB13560331; MiFish 12S, SUB13563458)  with BioProject number PRJNA985779."



## Results



## Discussion



## Acknowledgements



## Author contributions

## Conflict of interest statement

We declare no conflicts of interest.

## Reference list

::: {#refs}
:::
